src/postgres-util/Cargo.toml

```diff
@@ -35,8 +35,8 @@ workspace-hack = { version = "0.0.0", path = "../workspace-hack", optional = tru
 
 [features]
 default = ["workspace-hack", "replication", "schemas", "tunnel"]
-replication = ["tunnel"]
 schemas = ["prost", "serde", "proptest", "mz-proto", "tunnel"]
+replication = ["tunnel"]
```

Comment:
Did this get auto-sorted into this order?

Reply:
Nope, this was my mistake when resolving a merge conflict - switching back!